src/biokbase/narrative/exception_util.py

```diff
@@ -3,6 +3,22 @@
 from biokbase.userandjobstate.baseclient import ServerError as UJSServerError
 
 
+class NoJobException(ValueError):
```

Comment:
For more control when picking up exceptions from up the call stack, especially when JobComm needs to know what message to send the FE regarding an error